src/.vuepress/components/MkAd.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-<a :href="ad.link" target="_blank"><img :src="ad.img"></a>
+<a :href="ad.link" target="_blank"><img :src="ad.img" alt="Misskey ad from Ai-chan!"></a>
```

Comment:
`ad.img` isn't an advertisement for monetizing.

Suggestion:
Ai-chan! → Message from Ai-chan Misskey Official Mascot character

Reply:
Since this would need refactoring of a separate component I think it would be best to create a separate issue or PR for it.

Reply:
I agree. Creating a separate issue or PR is needed.